canonical-web-and-design/ubuntu.com
# Cube Study

## Done
Add study materials under `/cube/study`

## QA
- Check `https://ubuntu-com-9703.demos.haus/cube/study` is behind auth
- Check the all the navigation links works
- Check dynamic pagination follow the right order
- Check that `https://ubuntu-com-9703.demos.haus/core/smartstart/guide` is still alive as I made a change for it to have it's own template, since as it stood we here adding the smartstart marketo form to non smartstart pages.

**For the review/QA all the markdown content has been produced and reviewed by the CUBE team**


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3977
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3978
